assets/src/styles/blocks/Articles.scss

```diff
@@ -68,55 +77,47 @@
 
   @include medium-and-up {
     flex-basis: 30%;
-    order: 0;
-  }
-
-  @include large-and-up {
-    flex-basis: 30%;
   }
 }
 
 .article-list-item-body {
-  padding: $sp-2;
+  margin-top: $sp-1;
 
   @include medium-and-up {
+    margin-top: 0;
     flex-basis: 68%;
   }
-
-  @include large-and-up {
-    flex-basis: 68%;
-    padding: 0;
-  }
 }
 
 .article-list-item-headline {
   --block-articles--article--headline-- {
     font-weight: 500;
     color: $grey-80;
-    margin: 4px 0;
   }
 
   a {
     color: inherit;
   }
+
+  display: -webkit-box;
```

Comment:
I don't see why we need to add line clamping for the ticket's specs :thinking:

Reply:
[Updated](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/819/commits/21f4902c375143798521add6c5a4ad9476879408) to be only on mobile!